src/lib/schematics/install/index.spec.ts

```diff
@@ -41,7 +41,11 @@ describe('material-install-schematic', () => {
       './node_modules/@angular/material/prebuilt-themes/indigo-pink.css');
   });
 
-  it('should add custom theme', () => {
+  it('should support adding a custom theme', () => {
+    // TODO(devversion): currently a "custom" theme does only work for projects using SCSS.
```

Comment:
Does the CLI not support mixing scss with other preprocessors?

Reply:
It does. The problem with the `install` schematic right now is that we also add the custom theme code to projects that don't use SCSS.

So, SASS specific code will be added to the `styles.css` file, which results in a non-working application.